Patches/Forgelings race/Apparel_Forge.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+
+  <Operation Class="PatchOperationFindMod">
+    <mods>
+      <li>Forgelings race</li>
+    </mods>
+
+    <match Class="PatchOperationSequence">
+      <operations>
+
+      <!-- === Crusader Helmet === -->
+      <li Class="PatchOperationReplace">
+        <xpath>Defs/ThingDef[defName="Forge_Apparel_Helmet"]/statBases/StuffEffectMultiplierArmor</xpath>
+        <value>
+          <StuffEffectMultiplierArmor>3</StuffEffectMultiplierArmor>
```

Comment:
I might increase the thickness to 3.5 - 5, as it's a bulky, semi-industrial level tech of armor. In general, I think the armor should be able to stop pistol fire like 9mm.